gamedata/scripts/magazines.script

```diff
@@ -52,6 +52,35 @@ end
 --------------------------------------
 -- SECTION mag un/loading functions --
 --------------------------------------
+
+
+-- Get the time delay between loading/unloading rounds from/to magazine
+--
+-- **mag_section** *(string)* = section of the mag to get **load_delay** param if **ignore_loaddelay** is false 
+--
+-- **is_unload** *(bool)* = should we get mag unload time
+function get_mag_time(mag_section,is_unload)
+    is_unload=is_unload or false
+
+	local load_factor=magazines_mcm.get_config('mag_loadtime_factor')
+	local unload_factor=magazines_mcm.get_config('mag_unloadtime_factor')
+	local default_delay=0.4
+
+	if magazines_mcm.get_config('ignore_loaddelay') then
+		default_delay=1
```

Comment:
remove this line.
Without changing anything else checking this box now slows (un)loading by more than 2x.

Reply:
This is intended behaviour, it is stated in the checkbox desc that _factor_ setting should be treated as seconds if it's checked.
With the proposed changes, it should be mentioned that the constant value is 0.4 second and factor is applied to that

Reply:
OK. thats fine then.
Will still take some ppl by surprise as ppl don't read descriptions in my experience.  (obvs i didn't dig thru the xml that closely. srry)